documentation/modules/proc-configuring-kafka-connect-user-authorization.adoc

```diff
@@ -26,7 +26,7 @@ NOTE: The default values for the consumer group and topics will differ when xref
 
 . Edit the `authorization` property in the `KafkaUser` resource to provide access rights to the user.
 +
-For example, here we see access rights configured for the Kafka Connect topics and consumer group using the following `literal` name values:
+For example, here access rights are configured for the Kafka Connect topics and consumer group using the following `literal` name values:
```

Comment:
This reads a bit awkwardly now (sorry!). Why not say something like:

"In the following example, access rights are configured..."